src/world/common/UnkFunc50.inc.c

```diff
@@ -0,0 +1,11 @@
+#include "common.h"
+#include "map.h"
+
+ApiStatus N(UnkFunc50)(Evt* script, s32 isInitialCall) {
+    f32 temp_f2 = UNK_FUNC_50_VAR1 - script->varTable[0]; //-469.0
+    f32 temp_f0 = UNK_FUNC_50_VAR2 - script->varTable[2]; //46.0
```

Comment:
Remove these commands. Also these are x and z

Reply:
Ah i bet UNK_FUNC_50_VAR1 and UNK_FUNC_50_VAR2 are the X and Z of a cutscene trigger